client/ayon_core/plugins/publish/extract_review.py

```diff
@@ -881,49 +972,184 @@ def ffmpeg_full_args(
 
         return all_args
 
-    def fill_sequence_gaps(self, files, staging_dir, start_frame, end_frame):
-        # type: (list, str, int, int) -> list
+    def fill_sequence_gaps_with_previous(
+        self,
+        collection: str,
+        staging_dir: str,
+        instance: pyblish.plugin.Instance,
+        current_repre: Dict[Any, Any],
```

Comment:
I would suggest to pass only representation name (as that's the only thing actually being used).